Changelog.md

```diff
@@ -11,6 +11,7 @@ Breaking Changes:
  * Type Checker: Disallow arithmetic operations for Boolean variables.
  * Disallow trailing dots that are not followed by a number.
  * Remove assembly instructions ``sha3`` and ``suicide``
+ * Removed the obsolete files in the ``std`` directory.
```

Comment:
Should be ``remove`` :) Perhaps ``Remove obsolete std directory from project.`` is even better?

Reply:
Is it better this way?